rust-runtime/aws-smithy-http-server/src/operation/mod.rs

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Excellent documentation, thanks for writing this.

We will certainly reuse this in the public-facing Rustdocs / book / whatever we choose.